### PR TITLE
doc: release-notes-2-3: RISC-V related changes

### PR DESCRIPTION
**Pull request by PiotrZierhoffer**
I tried to sum up everything that's important in RISC-V arch and related boards. 
I also mention the accelerometer we added.

### CODE REVIEW

**Comment by carlescufi** [comment]
@PiotrZierhoffer could you rebase please?

**Comment by PiotrZierhoffer** [comment]
@carlescufi done

**Review by mgielda — APPROVED** [review]
LGTM

File: doc/releases/release-notes-2.3.rst [hunk 1]
```diff
@@ -187,6 +189,12 @@ Drivers and Sensors
 * Bluetooth
 
   * Add an RX thread on stm32wb hci wrapper
+  * Improve BLE support for rv32m1_vega:
+
+    - Add Resolvabl Private Address support
```
Comment (mgielda) [inline doc/releases/release-notes-2.3.rst:194]: just noticed - Resolvable?

Comment (PiotrZierhoffer) [reply doc/releases/release-notes-2.3.rst]: fixed

**Review by carlescufi — APPROVED** [review]
Thank you!

**Pull request merged**